vital/types/feature_track_set.cxx

```diff
@@ -43,6 +43,32 @@
 namespace kwiver {
 namespace vital {
 
+typedef std::unique_ptr<track_set_implementation> tsi_uptr;
+
+
+/// Default Constructor
+feature_track_set
+::feature_track_set()
+  : track_set(tsi_uptr(new simple_track_set_implementation))
+{
+}
+
+
+/// Constructor specifying the implementation
+feature_track_set
+::feature_track_set(std::unique_ptr<track_set_implementation> impl)
+  : track_set(std::move(impl))
```

Comment:
It certainly could, but I'm not sure that it needs too.  This signature matches the one in the header.  I only added the typedef because the first constructor required an explicit cast from raw to unique_ptr and writing it out on one line was a bit unwieldy.  Note that the typedef is local to this source file and not in the header.